graphql_api/types/upload/upload.graphql

```diff
@@ -26,5 +26,5 @@ type UploadErrorsEdge {
 }
 
 type UploadError {
-  errorCode: UploadErrorEnum
+  errorCode: UploadErrorCode
```

Comment:
I believe renames are considered breaking changes in GraphQL. Are we okay with such a "breaking change"?

Reply:
i'll revert, thanks for the callout!